src/libawkward/array/ListOffsetArray.cpp

```diff
@@ -227,7 +227,7 @@ namespace awkward {
       int64_t len = offsets_.length() - 1;
       Index64 out(len + 1);
       struct Error err =
-        util::awkward_listoffsetarray_compact_offsets64<int64_t>(
+        kernel::listoffsetarray_compact_offsets64<int64_t>(
```

Comment:
Similar to above, both the kernel dispatch and underlying kernel need an underscore in their names: `..._compact_offsets_64`.